### PR TITLE
Stage /startpos

### PR DESCRIPTION
**Pull request by shipyy**

### CODE REVIEW

**Comment by shipyy** [comment]
ignore this xD, my brain very 1iq

File: addons/sourcemod/scripting/surftimer/sql.sp [hunk 1]
```diff
@@ -11547,21 +11547,80 @@ public void SQL_viewCCP_GetMapRankCallback(Handle owner, Handle hndl, const char
...
+
+	Handle pack = CreateDataPack();
+	WritePackCell(pack, client);
+	WritePackString(pack, szSteamID);
+	WritePackString(pack, szMapName);
+	WritePackFloat(pack, map_time);
+	WritePackCell(pack, map_rank);
+	WritePackString(pack, szMapTimeFormatted);
+
+	Format(szQuery, 1024, "SELECT count(runtimepro)+1 FROM ck_playertimes WHERE mapname = '%s' AND style = '0';", szMapName);
```
[the remark this answers is not in the document]
Comment (shipyy) [reply addons/sourcemod/scripting/surftimer/sql.sp:11581]: i did went and got every format that uses "szQuery" to use sizeof() (not just in my changes :D ), but pr might not be stable yet for a submit xD

[the remark this answers is not in the document]
Comment (Bara) [reply addons/sourcemod/scripting/surftimer/sql.sp:11581]: I'm aware of it, if I remember me correct I changed this in the oldDev branch... but wasn't really "stable" and usable due some changes

[the remark this answers is not in the document]
Comment (shipyy) [reply addons/sourcemod/scripting/surftimer/sql.sp:11581]: i mean the pr is not only the stage /startpos , its a few other stuff :)

**Pull request closed without merging**